include/z64audio.h

```diff
@@ -784,7 +821,7 @@ typedef struct {
     /* 0x0C */ u8* curRamAddr;
     /* 0x10 */ u8* ramAddr;
     /* 0x14 */ s32 status;
-    /* 0x18 */ s32 bytesRemaining;
+    /* 0x18 */ u32 bytesRemaining;
```

Comment:
```suggestion
    /* 0x18 */ size_t bytesRemaining;
```
size_t is to express a count in size of bytes.